packages/@react-native-windows/telemetry/src/test/sanitize.test.ts

```diff
@@ -244,8 +244,8 @@ test('thrown exception a->b, hello world', done => {
 
       const filename = basename(__filename);
       expect(stack[0].method).toEqual('b');
-      expect(stack[1].method).toEqual('b');
-      expect(stack[2].method).toEqual('Object.a');
```

Comment:
@asklar : Just FYI: Updating some dependencies changed this. To me values seem that they were wrong before and correct what I would have expected now.